api/operate/docs/by-key-1.api.mdx

```diff
@@ -0,0 +1,61 @@
+---
+id: by-key-1
+title: "Get process instance by key"
+description: "Get process instance by key"
+sidebar_label: "Get process instance by key"
+hide_title: true
+hide_table_of_contents: true
+api: eJzlVktv2zgQ/isET304UdzN7sEoFnATNXCTJobtZA9BsKClsc1GIrUk5axX0H/vDPWoYzmB91g0h5gi55vnR84U3Iml5YN7PjY6AmtHyjqhIuAPPa4zMMJJrUYxH/D55hI2f/d5j2fCiBQcGAIWXOEHnj/CBs+kwmUm3ArXMdjIyIw04CaimV6wrLLDZGOoxw38k0sDaMSZHHrcRitIBR8U3G0yUi2VgyUYFF1okwpXbf1xysvygeA208qCJcSHkxP6eW56mkdkE/GRRlXKkYjIskRGPr7gmyW5omtZz79B5ChmQ9lwsrJCsR7inseR6TtMlqxsvAr67QOB5lmqmnrEWxjrjFRLr1YYDOPycDe8/OdEP13rGJoiH45HeePOhYO93oCKXzxD5L4TAuUpEW94NhvdhbhxdvN1fBXOwnNaD6/PwitcPvzI4TkspJJUr8P9xmIL5V5N4g7xvW4eSRPliTBvdk7feqXSJaRp98qU+Nfjp/sYOFJrkciYEdXBupeZiLHOE0jf/19GUppzeyC/UvRaLPfXq72X+w6rjc7BVk5CY7RpM9HvZuJWidyttJH/4YV/loZ3wbufP+LTbsSTquZA1bc6NxEwpR1b6FzFvwYRft93JYbjEdsKmIEH/AL5oEcRotxIt/H9cw74Dpkj31Pu8b0rMAX6UYL/ethtoxfgOj2UzTesar/YlleauvUSfCqoEQ94sO4HNeaowdigQEjJyRmzbnp5bhLql+HkLpwc3U6uOl38AhQNBUjmCsYIQV42IU2pLlXqmzDatIhMXno365HhZhxOhrPwaBpOp6Ob62Z8qHHUCLdS02pZOZfxekqg70qIN9Kfm6p++WvmKyDVQnt4XYkbP9YAG+dzZBdDHnainGkm/MiAP06ugQkVs0inWQIU+m7+acUatcgqlmpsVJo44JHOaLQFdqW1q5ofkRxVk1tVyikoOwiCp6en40ik+DSIYzRISUAnAccbkq3zdlXv9HbAsY5si5bafwcGFoB9LoKgVmQD0rpuJhLePz45PvH9UFuXCrVl6HWyPctZWx4H/7ogSwSWsqwdLGoi3vN1n7f9/AcVcW9AKpFGK/SBBItiLizcmqQsaRtfUOOvBzoujBRzqiTyNZaW1sj4hUgsdHxqHxP+ZlJPmW/Zy5Po3hAa8iqKGdt4Tl+49Kz0/0uaUlYgYqQheVWdDJFBmdvCdCZOujftjb0IZyhLzXErmTvc9tr3+vPxkxdgM/0I6s/WO0ef5F9ZfgcaSRP/
```

Comment:
Whats the purpose of this field and ss there something sensitive here that allows access to a live server? Or just a dummy text that can't be decrypted by anyone?

Reply:
I'm not exactly sure what it includes, but it is something serialized by the OpenAPI plugin that is needed to render the explorer properly. I'm not concerned about it containing sensitive info.